tests/Doctrine/Tests/ORM/Persisters/JoinedSubclassPersisterTest.php

```diff
@@ -37,4 +43,21 @@ public function testExecuteInsertsWillReturnEmptySetWithNoQueuedInserts(): void
     {
         self::assertSame([], $this->persister->executeInserts());
     }
+
+    public function testNonInsertablePropertyInChildClass(): void
```

Comment:
I would like this to be a functional test instead. We want to slowly move away from these mock tests to functional tests.

The entity should also be in the `Upsertable` model space, best to add one InsertableChild and InsertableParent there, and then move the test to `InsertableUpdateableTest` instead.